converter/converters/koki_pod_to_kube_v1_pod.go

```diff
@@ -68,7 +68,7 @@ func Convert_Koki_Pod_to_Kube_v1_Pod(pod *types.PodWrapper) (*v1.Pod, error) {
 	}
 	kubePod.Spec.RestartPolicy = restartPolicy
 
-	affinity, err := revertAffinity(kokiPod.Affinity)
+	affinity, err := Convert_Koki_Affinity_to_Kube_v1_Affinity(kokiPod.Affinity)
```

Comment:
Why an exported method? It's not a top level kubernetes resource type...

Reply:
Did it so the file looks like the others. Doesn't make a difference to me either way.

Food for thought on public vs private in library design:
http://youtu.be/yFXzuCFeRGM?t=1h34m55s